scipy/interpolate/fitpack.py

```diff
@@ -50,72 +49,79 @@ def _intc_overflow(x, msg=None):
     return intc(x)
 
 
-_iermess = {0:["""\
-    The spline has a residual sum of squares fp such that abs(fp-s)/s<=0.001""",None],
-               -1:["""\
-    The spline is an interpolating spline (fp=0)""",None],
-               -2:["""\
+_iermess = {0: ["""\
+    The spline has a residual sum of squares fp such that """
+                """abs(fp-s)/s<=0.001""", None],
+            -1: ["""\
+    The spline is an interpolating spline (fp=0)""", None],
+            -2: ["""\
```

Comment:
While you're cleaning these: ``["""\`` should really just be ``["`` and the string should start on the same line as ``["``.

Also the indent of -1 and -2 still doesn't line up with that of 0.